aspnet/includes/identity/identity_replace_membership.md

```diff
@@ -0,0 +1,8 @@
+> [!NOTE]
+> Since this article was written, the ASP.NET Membership providers have been superseded by [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity). We strongly recommend updating apps to use the [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity) platform rather than the Membership providers featured at the time this article was written. ASP.NET Identity has a number of advantages over the ASP.NET Membership system, including :
+>
+> * More secure.
+> * More preformant.
+> * Improved testability.
+> * Support for two-factor authentication.
+> * More features.
```

Comment:
@StephenMolloy  this is the only place the not will occur. I let the GitHub ChatGPT generate the note and it seemed pretty good. Can you make it better? We should probably give them some motivation to update.